Libplanet/Action/Context.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Runtime.CompilerServices;
+using Libplanet.Blocks;
+
+[assembly: InternalsVisibleTo("Libplanet.Tests")]
```

Comment:
Minor, but InternalsVisibleTo is a global setting so you do not have to declare it again.